tests/image/test_build.py

```diff
@@ -91,12 +92,12 @@ def test_cli_with_progress(docker_client_mock, project_dir, test_mock):
     result = runner.invoke(cli, ["dev"])
     assert result.exit_code == 0
     expected = format_expected_progress(
-        'Building registry.io/user/project:1.0.'
-        'Steps:   0%|          | 0/1'
-        'Steps: 100%|██████████| 3/3'
-        'The image registry.io/user/project:1.0 build is completed.'
+        r'Building registry\.io/user/project:1\.0\.'
+        r'Steps:   0%\| +\| 0/1'
+        r'Steps: 100%\|█+\| 3/3'
+        r'The image registry\.io/user/project:1\.0 build is completed\.'
```

Comment:
`invoke` above allows specifying a terminal width, but this seemed easier than to find the right width for some reason. But I can try the width approach if you prefer.